src/components/renderer/form-nested-screen.vue

```diff
@@ -0,0 +1,95 @@
+<template>
+  <vue-form-renderer
+    class="form-nested-screen"
+    :placeholder="placeholder"
+    v-model="data"
+    :config="config"
+    mode="preview"
+    :computed="computed"
+    :custom-css="customCSS"
+    :watchers="watchers"
+    v-on:css-errors="cssErrors = $event"
```

Comment:
In almost all places in the app, we use the `@` shorthand for attaching event listeners. Can you change this to:
```suggestion
    @css-errors="cssErrors = $event"
```